perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue

```diff
@@ -102,10 +100,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
             </v-hover>
           </div>
           <v-btn
-            v-if="displayBuyButton"
-            :disabled="disabledBuy || !walletEnabled || walletLoading"
+            :disabled="(disabledBuy || !walletEnabled || walletLoading) || !displayBuyButton"
             :loading="!disabledBuy && walletLoading"
-            :title="$t('exoplatform.perkstore.button.buy')"
+            :title="displayBuyButton ? $t('exoplatform.perkstore.button.buy') : $t('exoplatform.perkstore.button.disabledBuyButton')"
```

Comment:
You need to :
 - Use a computed here too
 - Add a message for each cause for which the button is disabled

Reply:
done !